PhotoBooth/TimeIntervalStepperViewModel.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+class TimeIntervalStepperViewModel: StepperViewModel {
+
+    var currentValue: TimeInterval
+
+    var minValue: Double = 1
+
+    var maxValue: Double = 10
```

Comment:
make variables that dont need to be exposed into `private` and reduce setter access of the other ones with `private(set) var` if you it to be `readonly`